app/js/actions/drug.js

```diff
@@ -1,24 +1,34 @@
 import axios from 'axios';
-import * as actionTypes from './actionTypes';
+import {
+  SEARCH_DRUGS_LOADING,
+  SEARCH_DRUGS_FAILURE,
+  SEARCH_DRUGS_SUCCESS,
+  SELECT_DRUG,
+} from './actionTypes';
 
 const contextPath = window.location.href.split('/')[3];
 const apiBaseUrl = `/${contextPath}/ws/rest/v1`;
```

Comment:
These two lines have been abstracted to a separate file to avoid code duplication. Could you import the `apiBaseUrl` file instead of re-writing it again?

Reply:
okay, lemme do that then. Thanks

Reply:
Done!